packages/contracts-bedrock/scripts/libraries/DeployUtils.sol

```diff
@@ -364,4 +364,23 @@ library DeployUtils {
             require(val == type(uint8).max, "DeployUtils: storage value is not 0xff at the given slot and offset");
         }
     }
+
+    /// @notice Etches a contract, labels it, and allows cheatcodes for it.
+    /// @param _etchTo Address of the contract to etch.
+    /// @param _cname The contract name (also used to label the contract).
+    /// @param _artifactPath The path to the artifact to etch.
+    function etchLabelAndAllowCheatcodes(address _etchTo, string memory _cname, string memory _artifactPath) internal {
+        vm.etch(address(_etchTo), vm.getDeployedCode(_artifactPath));
+        vm.label(address(_etchTo), _cname);
+        vm.allowCheatcodes(address(_etchTo));
```

Comment:
The extra typecast is redundant

Reply:
good catch. Weird that solidity doesn't complain about this. e6d771a8a539caffac72f232c3923b0da66f0530